src/oss/python/integrations/document_loaders/azure_blob_storage.mdx

```diff
@@ -0,0 +1,139 @@
+
+## Load from container
+
+```python
+loader = AzureBlobStorageLoader(
+    account_url="https://<storage-account-name>.blob.core.windows.net",
+    container="<container-name>",
+)
+
+for doc in loader.load():
+    print(doc)
+```
+
+```output
+[Document(page_content='Lorem ipsum dolor sit amet.', lookup_str='', metadata={'source': '/var/folders/y6/8_bzdg295ld6s1_97_12m4lr0000gn/T/tmpaa9xl6ch/fake.docx'}, lookup_index=0)]
+```
+
+You can also specify a prefix for more fine-grained control over what files to load.
+
+```python
+loader = AzureBlobStorageLoader(
+    account_url="https://<storage-account-name>.blob.core.windows.net",
+    container="<container-name>",
+    prefix="<prefix>",
+)
+```
+
+## Load from container by blob name
+
+You can load documents from a list of blob names, which uses only the blobs provided instead of a call to list blobs.
+
+```python
+loader = AzureBlobStorageLoader(
+    account_url="https://<storage-account-name>.blob.core.windows.net",
+    container_name="<container-name>",
+    blob_names=["blob-1", "blob-2", "blob-3"],
+)
+```
+
+## Override default credentials
+
+Below shows how to override the default credentials used by the document loader:
+
+```python
+from azure.core.credentials import AzureSasCredential
+from azure.identity import ManagedIdentityCredential
+from langchain_azure_storage.document_loaders import AzureBlobStorageLoader
+
+# Override with SAS token
+loader = AzureBlobStorageLoader(
+    "https://<storage-account-name>.blob.core.windows.net",
+    "<container-name>",
+    credential=AzureSasCredential("<sas-token>")
+)
+
+# Override with more specific token credential than the entire
+# default credential chain (e.g., system-assigned managed identity)
+loader = AzureBlobStorageLoader(
+    "https://<storage-account-name>.blob.core.windows.net",
+    "<container-name>",
+    credential=ManagedIdentityCredential()
+)
+```
+
+## Customize blob content parsing
+
+Currently, the default when parsing each blob is to return the content as a single `Document` object with UTF-8 encoding regardless of the file type. For file types that require specific parsing (e.g., PDFs, CSVs, etc.) or when you want to control the document content format, you can provide the `loader_factory` argument to take in an already existing document loader (e.g., PyPDFLoader, CSVLoader, etc.) or a customized loader.
+
+This works by downloading the blob content to a temporary file. The `loader_factory` then gets called with the filepath to use the specified document loader to load/parse the file and return the `Document` object(s).
+
+Below shows how to override the default loader used to parse blobs as PDFs using the [PyPDFLoader](https://python.langchain.com/api_reference/community/document_loaders/langchain_community.document_loaders.pdf.PyPDFLoader.html#pypdfloader):
+
+```python
+from langchain_azure_storage.document_loaders import AzureBlobStorageLoader
+from langchain_community.document_loaders import PyPDFLoader
```

Comment:
Let's maybe add an inline comment that it requires `langchain-community` and `pypdf` .